tripy/tests/frontend/test_cache.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import os
+
+import tripy as tp
+
+from tripy.frontend.trace import Trace
+from tripy.frontend.cache import ExecutableCache
+
+
+@pytest.fixture
+def cache():
+    return ExecutableCache()
+
```

Comment:
Can we just inline this in `mock_global_cache`?

Reply:
done

Reply:
No I mean can we just write:
```py
@pytest.fixture
def mock_global_cache(monkeypatch):
    cache = ExecutableCache()
    monkeypatch.setattr(tp.frontend.cache, "global_cache", cache)
    return cache
```
? I don't see a reason to have a separate fixture that's only used in one place.

Reply:
i see done